README.md

```diff
@@ -204,6 +204,41 @@ service.metadata.labels.role = 'slave'
 client.create_service service`
 ```
 
+Below example is for the extension API
```

Comment:
@henders IIRC there are other places in the README that should be updated to mention deployments.